src/pages/angular/virtual-scroll.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Virtual Scroll in Angular'
+---
+
+In previous versions of Ionic, there had been a `virtual-scroll` directive to help with list virtualization. But as of recent, Angular provides it's own solution with the `@angular/cdk` package.
+
+## Install
```

Comment:
```suggestion
## Setup
```

Maybe call this setup? I'm thinking for other frameworks there may not be an "Install" option